akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/Behaviors.scala

```diff
@@ -30,7 +30,7 @@ object Behaviors {
 
   /**
    * `deferred` is a factory for a behavior. Creation of the behavior instance is deferred until
-   * the actor is started, as opposed to `Actor.immutable` that creates the behavior instance
+   * the actor is started, as opposed to [[immutable]] that creates the behavior instance
```

Comment:
I think the conclusion was that this will not work with genjavadoc, it has to be `Behaviors#immutable` or `#immutable`, and intellij is complaining about the latter

Reply:
Thanks